C/plugins/storage/postgres/init.sql

```diff
@@ -45,42 +45,30 @@
 -- SCHEMA CREATION
 ----------------------------------------------------------------------
 
+-- Disable the NOTICE notes
+SET client_min_messages TO WARNING;
 
 -- Dropping objects
 DROP SCHEMA IF EXISTS foglamp CASCADE;
 DROP DATABASE IF EXISTS foglamp;
 DROP TABLESPACE IF EXISTS foglamp;
-DROP USER IF EXISTS foglamp;
-
--- Create the foglamp user
-CREATE USER foglamp WITH
-  LOGIN
-  SUPERUSER
-  INHERIT
-  CREATEDB
-  CREATEROLE
-  REPLICATION;
-
-ALTER USER foglamp ENCRYPTED PASSWORD 'foglamp';
 
 
 -- Create the foglamp tablespace
 CREATE TABLESPACE foglamp
   OWNER foglamp
-  LOCATION '/var/lib/postgresql/9.6/';
+  LOCATION '/usr/local/foglamp/data/storage/postgres';
```

Comment:
What if we have not set FOGLAMP_HOME to /usr/local? Can we make this work with FOGLAMP_HOME? Or in fact should it not be FOGLAMP_DATA? Thinking in particular of how this would work in a snap package.

Reply:
Sure, I thought about that. 
In this first version, the plugin works with Linux user foglamp and FOGLAMP_ROOT: /usr/local/foglamp (so FOGLAMP_DATA: /usr/local/foglamp/data)
The CREATE TABLESPACE command accepts only an absolute path, therefore the package installer must change it in a post-install.
There are many options to avoid this, but they are fairly complicated, I would suggest to keep it simple for the moment and leave it like this, then change it in one of the next sprints.

Reply:
Sure, let's add a Jira for a future sprint